docs/configuring.md

```diff
@@ -996,6 +996,68 @@ on them, so they are always installed when Copier is installed.
     [other Jinja2 topics](https://github.com/search?q=jinja&type=topics), or
     [on PyPI using the jinja + extension keywords](https://pypi.org/search/?q=jinja+extension).
 
+### `message_after_copy`
+
+-   Format: `str`
+-   CLI flags: N/A
+-   Default value: `""`
+
+A message to be printed after _successful_ [project creation](../generating).
+
+If the message contains Jinja code, it will be rendered with the same context as the
+rest of the template. A [Jinja include](#importing-jinja-templates-and-macros)
+expression may be used to import a message from a file.
+
+The message is suppressed when Copier is run in [quiet mode](#quiet).
+
+!!! example
+
+    ```yaml title="copier.yml"
+    project_name:
+        type: str
+        help: An awesome project needs an awesome name. Tell me yours.
+
+    _message_after_copy: |
+        Your project "{{ project_name }}" has been created successfully!
+
+        Next steps:
+
+        1. Change directory to the project root:
+
+           $ cd {{ _copier_conf.dst_path }}
+
+        2. Read "CONTRIBUING.md" and start coding.
+    ```
+
+### `message_before_copy`
+
+-   Format: `str`
+-   CLI flags: N/A
+-   Default value: `""`
+
+A message to be printed before [project creation](../generating).
+
+If the message contains Jinja code, it will be rendered with the initial context which
+does not yet contain any answers to questions. A
+[Jinja include](#importing-jinja-templates-and-macros) expression may be used to import
+a message from a file.
+
+The message is suppressed when Copier is run in [quiet mode](#quiet).
```

Comment:
I think you could refer to one section from the other, to DRY.

Reply:
Done.